### ISSUE TITLE
CA1823: Fields used in attributes are being incorrectly identified as unused

### ISSUE THREAD

**Issue by srivatsn** [dotnet/roslyn-analyzers#1222]
#### Analyzer

CA1823: Unused fields - 2.3.0-beta1

#### Repro steps

```csharp
   [ClassRegistration(DebugPropertyClassId, DebugPropertyClassInfo)]
    internal class CSharpProjectSystemPackage : AsyncPackage
    {
        private const string DebugPropertyClassId = "{0273C280-1882-4ED0-9308-52914672E3AA}";
        private const string DebugPropertyClassInfo 
```

#### Expected behavior
No warnings

#### Actual behavior
DebugPropertyClassId and DebugPropertyClassInfo are identified as unused.

**Comment by mavasani** [comment]
I think Fred already has a tracking bug for this.

**Comment by 333fred** [comment]
Yep, https://github.com/dotnet/roslyn-analyzers/issues/1191.
